uwsgi.h

```diff
@@ -1515,8 +1522,9 @@ struct wsgi_request {
 	// when set, do not send warnings about bad behaviours
 	int post_warning;
 
-	size_t range_from;
-	size_t range_to;
+	enum uwsgi_range range_parsed;
+	int64_t range_from;
```

Comment:
please move new struct fields at the end of the struct to retain ABI compatibility